projects/find-your-way/index.md

```diff
@@ -0,0 +1,108 @@
+The main task of the project was to provide a set of instructions in a human language emulating algorithms in machines. The instructions had to be in one sentence, simple to understand and could be composed with parameters or constants.
+
+## Algorithm part 1.
+
+
+![](./assets/images/algo-1.png)
+
+
+#### _“Draw round shapes in vertical and horizontal which differ from the diameter. Create lines that merge into different points”._ 
+---
+
+
+The seminar began with the task of writing the first line of an analogue algorithm. The inspiration and consequent work flow are influenced by the results from a creative class mate, as well as the comprehension for the upcoming executed algorithm and the first steps in the project implementation.
+
+
+
+
+## Algorithm part 2.
+
+
+
+
+![](./assets/images/algo-2.png)
+
+
+#### _“Draw circles of various sizes throughout the sheet distant each other. Trace straight lines which intersect but do not touch the circles._ 
+
+---
+
+The second attempt for an analogue algorithm, was more accurate on the description to obtain a similar result of the previously choosen work. 
+
+Even if the instructions to be performed have been written more clearly, most of the results were out of every expectation. The purpose wasn't to create a text that scares the reader or makes him think too much for concluding an action. For that reason it was really important to phrase the instruction clear to prevent any misunderstandings.
+
+While looking on different drawings, it is recognizable that every person gives a different touch and express them self in different ways by just using a circle and a line. The choice of the material was free, which also included objects. 
+
+The interesting thing about the output was set by the different visual tension giving from a circle and a line to create a grid. The problems, instaed, was to cover the entire surface for each algorithm performed.
+
+## Algorithm part 3.
+
+
+![](./assets/images/algo-3.png)
+
+#### _Draw twenty filled circles of different sizes._
+#### _Circles connect with a straight line._ 
+#### _Connections must cross at least once._
+
+---
+
+The third part of the algorithm was set by an amount of circle to draw and a rule for crossing each line at least once. The material selected in advance was a brush and ink on rough paper. This will ensure a more controlled result at the end. 
+
+
+![](./assets/images/algo-4.png)
+
+
+By noting the individual result of the performers, it was immediately recognizable that it has to be more controlled by the material on which it runs. The choice was made by selecting the most suitable result of all the executed algorithm, which gave an idea for creating more visual depth and a percentage of circles, instead of a defined number of circles.
+
+
+
+## Conclusion
+
+![](./assets/images/algo-5.png)
+
+#### _Draw a circle consisting of 20 to 30 filled circles that do not touch each other._ 
```

Comment:
To many headlines. The ` ###`  will be transformed into HTML <h1234…> tags. These tend to not break and flow out of the page. Please find another way to highlight that.